addons/account_edi_proxy_client/models/account_edi_proxy_user.py

```diff
@@ -64,10 +65,25 @@ class AccountEdiProxyClientUser(models.Model):
 
     _sql_constraints = [
         ('unique_id_client', 'unique(id_client)', 'This id_client is already used on another user.'),
-        ('unique_edi_identification', 'unique(edi_identification, proxy_type, edi_mode)', 'This edi identification is already assigned to a user'),
-        ('unique_company_proxy', 'unique(company_id, proxy_type, edi_mode)', 'This company has a user already created for this EDI type'),
+        ('unique_active_edi_identification', '', 'This edi identification is already assigned to an active user'),
+        ('unique_active_company_proxy', '', 'This company has an active user already created for this EDI type'),
```

Comment:
Empty constraints?

Reply:
see https://github.com/odoo/odoo/pull/129595#discussion_r1275120223 :slightly_smiling_face:

Reply:
Yep I got it, but see https://github.com/odoo/odoo/pull/129595/files#r1277723160 ;)